src/create_atoms.cpp

```diff
@@ -968,7 +964,12 @@ int CreateAtoms::add_bisection(const double vert[3][3], tagint molid)
     if ((center[0] >= sublo[0]) && (center[0] < subhi[0]) && (center[1] >= sublo[1]) &&
         (center[1] < subhi[1]) && (center[2] >= sublo[2]) && (center[2] < subhi[2])) {
 
-      atom->avec->create_atom(ntype, center);
+      if (mode == ATOM) atom->avec->create_atom(ntype, center);
```

Comment:
Using the "mesh" option with molecule files makes no sense. These changes need to be reverted.